### ISSUE TITLE
write_compendium_description duplicates

### ISSUE THREAD

**Issue by JKRWard** [karthik/holepunch#24]
When running the `write_compendium_description` multiple times if I ask it to overwrite the pre-existing DESCRIPTION file package dependencies are being added each time so it's duplicating the list of packages. 
So working through the tutorial if I run the function three times I get the  the following in my DESCRIPTION file 

```
Type: Compendium
Package: Jess's-awesome-compendium
Title: What the Package Does (One Line, Title Case)
Version: 0.0.1
Authors@R: 
    person(given = "First",
           family = "Last",
           role = c("aut", "cre"),
           email = "first.last@example.com",
           comment = c(ORCID = "YOUR-ORCID-ID"))
Description: A wonderful compendium
License: What license it uses
Depends: 
    dataRetrieval,
    dataRetrieval,
    dataRetrieval,
    dplyr,
    dplyr,
    dplyr,
    ggplot2,
    ggplot2,
    ggplot2,
    lubridate,
    lubridate,
    lubridate,
    tidyr,
    tidyr,
    tidyr,
    viridis,
    viridis,
    viridis
Encoding: UTF-8
LazyData: true
```

**Comment by karthik** [comment]
Thanks so much for reporting this bug. I'll work on a fix asap.

**Comment by karthik** [comment]
Thanks so much for catching this bug. It's fixed now in `0.1.16.9000`.
